### DISCUSSION TITLE
3d printable file for Raspberry Pi Zero?

### DISCUSSION THREAD

**Discussion by seymourgler** [probonopd/MiniDexed#665] [Q&A]
Hello all, thank you again for sharing this awesome MiniDexed project. I love the 3d printable file, but was wondering if you've created a similar STL version just like that but for a Raspberry Pi Zero, an LCD1602, and a rotary encoder? Thanks!

**Comment by probonopd** [comment]
I had experimented with this but never really finished it so far.

![image](https://github.com/probonopd/MiniDexed/assets/2480569/e8bc26a6-f035-4743-b4c2-c742152fd362)

During my experiments, I fried one Raspberry Pi Zero due to some contacts touching each other due to the limited space in my housing. Thus, I have decided not to publish it. Also, got carried away with [another concept](https://github.com/probonopd/MiniDexed/discussions/323). Maybe I'll do a better one at some time - no promises though.
